src/components/whats-new.vue

```diff
@@ -60,12 +64,37 @@ watch(() => projects.dataExists, () => {
 });
 
 function hideModal() {
+  // Set modal to always be dismissed
   currentUser.preferences.site.whatsNewDismissed2025_1 = true;
+  // Set mailing list opt in to what was selected
+  currentUser.preferences.site.mailingListOptIn = mailingListOptIn.value;
```

Comment:
I'm thinking that we should only set this if there's been a change. Otherwise, it'll always send a request.

```js
if (mailingListOptIn.value !== currentUser.preferences.site.mailingListOptIn)
  currentUser.preferences.site.mailingListOptIn = mailingListOptIn.value;
```